src/frontend/screens/Settings/components/AutoDXVK.tsx

```diff
@@ -13,8 +12,8 @@ const AutoDXVK = () => {
     'autoInstallDxvk',
     false
   )
-  const home = configStore.get('userHome', '')
-  const [winePrefix] = useSetting('winePrefix', `${home}/.wine`)
+  const [defaultWinePrefix] = useSetting('defaultWinePrefix', '')
+  const [winePrefix] = useSetting('winePrefix', defaultWinePrefix)
```

Comment:
I have a question about this, I see `winePrefix` is having `defaultWinePrefix` as the fallback/default value

this default value before this change was `~/.wine` (a prefix), after this change it would be the value of the `defaultWinePrefix` config, but I see that is set here https://github.dev/Heroic-Games-Launcher/HeroicGamesLauncher/blob/690f522155940d2fc6dff3339a53f38002ee37d9/src/backend/config.ts#L609-L610 with the value of the `heroicDefaultWinePrefix` variable, which is `~/Games/Heroic/Prefixes` (the folder where we add prefixes, not a prefix), shouldn't we be using `~/Games/Heroic/Prefixes/default` here too instead?

Reply:
I think we probably want to use `/Games/Heroic/Prefixes/default` as the defaultWinePrefix fallback instead of `''`, a `''` as a value means that we could be setting an empty string as the winePrefix path in some edge case

Reply:
You are right. Anyway if the defaultWinePrefix is empty, someone really messed up because it is set by a constant, so i didn't added a fallback for the prefix directory.